descartes_light/test/src/perf_plotter.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+
+import math
+import numpy as np
+# import matplotlib as mpl
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import axes3d, Axes3D #<-- Note the capitalization! 
+import sys
+
+class Perf_Plotter():
+    def __init__(self):
+        self.dict = {}
+
+    def strip(self, filename = "../benchmarks/benchmarks.txt"):
```

Comment:
It probably makes sense for us to save the results of the benchmark into a structured format (like CSV, YAML, JSON, XML, etc.), rather than parsing the text that we arbitrarily wrote to document the benchmark results.

My vote would be to use YAML in a structure like this:
```yaml
- name: planner_name
  results:
    - n_waypoints: 10
      n_samples: 10
      dof: 6
      build_time: 0.05185
      search_time: 6.2715678
    - n_waypoints: 10
      n_samples: 50
      dof: 6
      build_time: 0.06768
      search_time: 8.123486
```

@Levi-Armstrong any thoughts on the structure of the benchmark output, especially if we plan to use Google Benchmark in the future?